api/client/connection.go

```diff
@@ -368,11 +370,18 @@ func (c *Connection) LogPostPath(ctx context.Context, space api.SpaceID, opts *L
 	return payloads.Error()
 }
 
-func (c *Connection) LogPostPathStream(ctx context.Context, space api.SpaceID, opts *LogPostOpts, paths ...string) (*Stream, error) {
-	body := api.LogPostRequest{Paths: paths}
-	if opts != nil {
-		body.JSONTypeConfig = opts.JSON
-		body.StopErr = opts.StopError
+func (c *Connection) LogPostPathStream(ctx context.Context, space api.SpaceID, opts LogPostOpts, paths ...string) (*Stream, error) {
+	body := api.LogPostRequest{
+		Paths:          paths,
+		JSONTypeConfig: opts.JSON,
+		StopErr:        opts.StopError,
+	}
+	if opts.Shaper != nil {
+		raw, err := json.Marshal(opts.Shaper)
+		if err != nil {
+			return nil, err
+		}
+		body.Shaper = raw
 	}
 	req := c.Request(ctx).
 		SetBody(body)
```

Comment:
Shouldn't the methods `LogPost`, `LogPostReaders` and `LogPostWriter` also handle the `Shaper` option?

Reply:
done in 085394f24905391eff81c5d9a26e0017284d9352